### ISSUE TITLE
`use_debounce` not available on crates.io latest version (0.5.1)

### ISSUE THREAD

**Issue by miguelpduarte** [DioxusLabs/sdk#65]
Title. crates.io has v`0.5.1` which does not include `use_debounce`, regardless of activated features, and despite the `main` branch including it, with a `Cargo.toml` that mentions version `0.5.0`.

Would it be possible to publish and updated release to include this hook? It is very useful for a use-case that I am facing right now.

**Comment by DogeDark** [comment]
We decided to wait to release `use_debounce` and the other new features until `0.6` to avoid breaking semver. SDK 0.6 will be released once Dioxus 0.6 releases and it's planned to come out soon.

In the meantime, you should be able to use the git main version of this repo to use `use_debounce`:
```toml
dioxus-sdk = { git = "https://github.com/dioxuslabs/sdk", features = ["timing"] }
```

**Comment by DogeDark** [comment]
SDK 0.6 has been released and `use_debounce` is now available on crates.io!
